include/bluetooth/bluetooth.h

```diff
@@ -279,6 +279,21 @@ int bt_le_scan_start(const struct bt_le_scan_param *param, bt_le_scan_cb_t cb);
  */
 int bt_le_scan_stop(void);
 
+/** @brief Helper for parsing advertising (or EIR) data.
```

Comment:
Add OOB

Reply:
In fact you could use the spec's language: "This part defines the basic data types used for Extended Inquiry Response
(EIR), Advertising Data (AD), and OOB data blocks."

Reply:
done

Reply:
oh, I missed your second comment. I'll update the documenation once more

Reply:
fixed now